src/cmd/cli/command/servicemonitor.go

```diff
@@ -16,14 +16,26 @@ func waitServiceStatus(ctx context.Context, targetStatus cli.ServiceStatus, serv
 	}
 
 	// set up service status subscription (non-blocking)
-	serviceStatusChan, err := cli.Subscribe(ctx, client, serviceList)
+	subscribeServiceStatusChan, err := cli.Subscribe(ctx, client, serviceList)
 	if err != nil {
 		term.Debugf("error subscribing to service status: %v", err)
 		return err
 	}
 
+	serviceStatus := make(map[string]string, len(serviceList))
+	for _, name := range serviceList {
+		serviceStatus[name] = string(cli.ServiceUnknown)
```

Comment:
that name-normalization logic messes with my brain; are we getting normalized or non-normalized names? I think we should remove the `normalizedServiceNameToServiceName` map and just make sure we internally only use the normalized ones. We can normalize any strings the users passes to use, either on cmd line or in compose.